### PR TITLE
fix: Bug in adding availability model, no extra space, cell, time appears

### PR DESCRIPTION
**Pull request by sophiagal**

### CODE REVIEW

File: src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx [hunk 1]
```diff
@@ -29,15 +29,36 @@ function AvailabilityScheduleSelector(
...
+            return <div />;
+        }
+
+        if (currentTime < endTime && Math.floor(currentTime) !== 0) {
+            return <div id={selectionId} ref={refSetter} />;
+        }
+        return <div />;
     };
 
+    //     time: Date, // being used internally, somehow -Jorge (laughed after I literally put what he said)
```
Comment (navarrojandg) [inline src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx:53]: Is there a reason why these lines are commented out?

Comment (ivancuadra869) [reply src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx:53]: I was there when Sophia was working on this. She deleted the commented-out lines, I think she just needs to push those changes perhaps.

Comment (navarrojandg) [reply src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx:53]: Okay -- typically, unless there is a good reason for leaving old code commented, it should be deleted

Comment (sophiagal) [reply src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx:53]: yeah, it was there before I started working on it. I'll delete it and commit again

**Review by navarrojandg — APPROVED** [review]
Just remove the commented old code, lgtm

**Review by jesc0bar — APPROVED** [review]

**Pull request merged**